docs/src/modules/java/pages/using-jwts.adoc

```diff
@@ -5,7 +5,7 @@ include::ROOT:partial$include.adoc[]
 
 This section describes the practical aspects of using JSON Web Tokens (JWTs) with the Java SDK, if you are not sure what JWTs are, how they work or how to generate them, see https://docs.kalix.io/security/jwts.html[JSON Web Tokens] first.
 
-Kalix’s JWT support is configured by placing annotations on methods in your endpoints.
+Kalix's JWT support is configured by placing annotations on methods in your endpoints or at the component type level.
```

Comment:
```suggestion
Kalix's JWT support is configured by placing annotations on methods in your endpoints or at the component class level.
```

Reply:
I have switched back and forth a few times. When defining an annotation, we need to say that it can be added to a type as oppose to a method or field. Hence the idea of referring to it as type-level annotations. 

But let's go with class. It's probably better.